README.md

```diff
@@ -25,13 +25,13 @@ A minimalistic Flutter game engine.
 
 ## About 1.0.0
 
-This is a release candidate for version 1.0.0 of Flame. This is not ready for production use and the documentation found here is not completely updated according to the code, also everything here can be changed before the full release.
+Our goal is to release v1 soon. We are periodically launching RCs (release candidates) as we evolve the code, and we are already quite happy with where we are (but there might be still further changes). Our documentation is still not completely updated to v1, and things are still open to change.
 
-Use this version to get a preview of how the new Flame version will be and also to give the team feedback about the new structure and/or features.
+Please use this version to get a preview of how the new Flame version will be and also to give the team feedback about the new structure and/or features.
```

Comment:
I would write:
Please use this version to get a preview of the new Flame version and also ...